deploy/kubernetes/external-health-monitor-controller/deployment.yaml

```diff
@@ -0,0 +1,51 @@
+# This YAML file demonstrates how to deploy the external
+# health monitor controller for use with the mock CSI driver. It
+# depends on the RBAC definitions from rbac.yaml.
+
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csi-external-health-monitor-controller
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      external-health-monitor-controller: mock-driver
+  template:
+    metadata:
+      labels:
+        external-health-monitor-controller: mock-driver
+    spec:
+      serviceAccount: csi-external-health-monitor-controller
+      containers:
+        - name: csi-external-health-monitor-controller
+          image: quay.io/k8scsi/csi-external-health-monitor-controller:canary
```

Comment:
We are going to move the image repo to a different place so images for this repo won't be here any more.
To avoid confusion, please change this to:
image: <csi-external-health-monitor-controller image place holder>

We can update this after this PR is merged and all necessary release-tools changes are merged.

Reply:
Can we merge this PR now?

Reply:
I'm still reviewing this.